repository/query/query.go

```diff
@@ -46,6 +46,10 @@ var (
 	DeleteMember string
 	//go:embed scripts/list_members_for_project.sql
 	ListMembersForProject string
+	//go:embed scripts/read_member.sql
+	ReadMember string
+	//go:embed scripts/read_member_by_email.sql
+	ReadMemberByEmail string
```

Comment:
Naming will be updated once the conclusion is reached here - https://github.com/jan-zabloudil/release-manager/pull/69#pullrequestreview-2059661739